getAccessToken.js

```diff
@@ -0,0 +1,14 @@
+const { getAccessTokenFromApiKey } = require('./utils/apiUtil');
+
+TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
+// console.log('target environment: ' + TARGET_ENVIRONMENT);
+API_KEY = process.env.API_KEY;
+// console.log('api key: ' + API_KEY);
+
+// obtain access token through fence API
+getAccessTokenFromApiKey(API_KEY, TARGET_ENVIRONMENT)
+    .then((ACCESS_TOKEN) => {
+	console.log('access_token: ' + ACCESS_TOKEN);
```

Comment:
Just output the token, so we can do things like: `token="$(getAccessToken.sh)"`

Reply:
That's clever.
Pushing a commit to address that.